DMX.Portal.Web/Services/Foundations/Labs/LabService.cs

```diff
@@ -23,6 +23,13 @@ public partial class LabService : ILabService
             this.loggingBroker = loggingBroker;
         }
 
+        public ValueTask<Lab> AddLabAsync(Lab lab) =>
+        TryCatch(async () =>
+        {
+            ValidateLabOnAdd(lab);
+            return await this.dmxApiBroker.PostLabAsync(lab);
```

Comment:
`return` statements should have their own extra empty line before them even if they are one-liners.